docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+        train, test = train_test_split(data)
+
+        # The predicted column is "quality" which is a scalar from [3, 9]
+        train_x = train.drop(["quality"], axis=1)
+        test_x = test.drop(["quality"], axis=1)
+        train_y = train[["quality"]]
+        test_y = test[["quality"]]
+
+        alpha = float(args.alpha)
+        l1_ratio = float(args.l1_ratio)
+
+        with mlflow.start_run():
+            lr = ElasticNet(alpha=alpha, l1_ratio=l1_ratio, random_state=42)
+            lr.fit(train_x, train_y)
+
+            predicted_qualities = lr.predict(test_x)
+
+            (rmse, mae, r2) = eval_metrics(test_y, predicted_qualities)
+
+            print("Elasticnet model (alpha=%f, l1_ratio=%f):" % (alpha, l1_ratio))
+            print("  RMSE: %s" % rmse)
+            print("  MAE: %s" % mae)
+            print("  R2: %s" % r2)
+
+            mlflow.log_param("alpha", alpha)
+            mlflow.log_param("l1_ratio", l1_ratio)
+            mlflow.log_metric("rmse", rmse)
+            mlflow.log_metric("r2", r2)
+            mlflow.log_metric("mae", mae)
+            mlflow.set_tag("test", '13')
+
+            mlflow.sklearn.log_model(lr, "model")
+
+            # Persist samples (input and output)
+            train_x.head().to_pickle('head_input.pkl')
+            mlflow.log_artifact('head_input.pkl', 'model')
+            train_y.head().to_pickle('head_output.pkl')
+            mlflow.log_artifact('head_output.pkl', 'model')
+
+In this file, we're:
+
+- Starting run context on line 46
+- Training ``ElasticNet`` model on line 48
+- Setting metrics, parameters and tags on lines 59-64
+- Saving (through serialization) model with name ``model`` on line 66
+- Saving input abd output samples (for persisting information about input and output column names) on lines 69-72
+
+
+Prepare Legion Platform connections
+------------------------------------
+
+As mentioned before Legion Platform uses concept of Connections to manage different kind of data and other external services
+To complete this tutorial we will need next connections
+
+- Connection to VCS repository where MLFlow project for wine classification is located
+- Connection to wine-quality.csv file in one of supported object storage
+- Connection to docker repo where out trained and packed model will be stored
+
+Let's create these connections. Legion examples repository already contains payload templates so we will use it
+
+Let's create connection to VCS repository that contains project code. Because legion-examples repository already contains required project
+we just create connection that points to this Git repository.
+
+.. code-block:: console
+
+    $ cat ./legion/vcs_connection.legion.yaml
+
+    kind: Connection
+    id: legion-examples
+    spec:
+      type: git
+      uri: git@github.com:legion-platform/legion-examples.git
+      reference: origin/master
+      publicKey: <paste here your public github ssh key>
```

Comment:
It's a optional field. In order not to overcomplicate, I would suggest removing this field.